.schemas/config.schema.json

```diff
@@ -1468,6 +1468,38 @@
               }
             }
           ]
+        },
```

Comment:
Revert, see https://github.com/ory/oathkeeper/blob/master/.schemas/README.md :) Changes should be made here: https://github.com/ory/oathkeeper/tree/master/.schema/